bundles/org.openhab.binding.heliosventilation/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,235 @@
+		<category>Temperature</category>
+		<state pattern="%d %unit%" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="supply_temperature" advanced="false">
+		<item-type>Number:Temperature</item-type>
+		<label>Supply Temperature</label>
+		<description>Temperature measured in the supply (incoming) air flow.</description>
+		<category>Temperature</category>
+		<state pattern="%d %unit%" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="outgoing_temperature" advanced="false">
+		<item-type>Number:Temperature</item-type>
+		<label>Outgoing Temperature</label>
+		<description>Temperature measured in the outgoing air flow.</description>
+		<category>Temperature</category>
+		<state pattern="%d %unit%" readOnly="true" />
+	</channel-type>
+
+
+	<channel-type id="set_temperature" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Set temperature for the supply air. Not used in all ventilation systems.</description>
+		<category>Temperature</category>
+		<state pattern="%d %unit%" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="fanspeed" advanced="false">
+		<item-type>Number</item-type>
+		<label>Fanspeed</label>
+		<category>HVAC</category>
+		<state pattern="%d" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="min_fanspeed" advanced="true">
+		<item-type>Number</item-type>
+		<label>Minimal Fanspeed</label>
+		<category>HVAC</category>
+		<state pattern="%d" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="max_fanspeed" advanced="true">
+		<item-type>Number</item-type>
+		<label>Maximum Fanspeed</label>
+		<category>HVAC</category>
+		<state pattern="%d" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="bypass_temperature" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Cell Bypass Temperature</label>
+		<description>Bypass temperature to disable the bypass function if outside temperature is below this threshold even if ventilation system is in summer mode.</description>
+		<category>Temperature</category>
+		<state pattern="%d %unit%" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="supply_stop_temperature" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Supply Stop Temperature</label>
+		<description>Stop the supply fan if outside temperature is below this threshold.</description>
+		<category>Temperature</category>
+		<state pattern="%d %unit%" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="preheat_temperature" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Preheat Temperature</label>
+		<description>Set temperature for preheater.</description>
+		<category>Temperature</category>
+		<state pattern="%d %unit%" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="rh_limit" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>RH limit</label>
```

Comment:
Please make sure to use camel case in all label texts - there are further occurrences to fix below.

```suggestion
		<label>RH Limit</label>
```